src/main/java/com/autosale/shop/controller/SecurityController.java

```diff
@@ -0,0 +1,55 @@
+package com.autosale.shop.controller;
+
+import com.autosale.shop.model.User;
+import com.autosale.shop.security.JwtTokenUtil;
+import com.autosale.shop.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequiredArgsConstructor
+public class SecurityController {
+
+    private final UserService userService;
+
+    private final JwtTokenUtil jwtTokenUtil;
```

Comment:
JwtTokenService? Util classes usually has some static method that not depends on Spring Context